charts/tidb-cluster/values.yaml

```diff
@@ -171,6 +171,35 @@ monitor:
   #   value: tidb
   #   effect: "NoSchedule"
 
+fullbackup:
+  create: false
+  binlogImage: pingcap/tidb-binlog:latest
+  mydumperImage: pingcap/tidb-enterprise-tools:latest
```

Comment:
Can we use a separate mydumper image? Particularly since mydumper is its own GPL licensed software and we are iterating to push our changes upstream. It would also remove the need to configure `mydumperBin`

Reply:
Removed `mydumperBin`.

Reply:
We can change the image to a separate mydumper image when it is available.

Reply:
:+1: I will make a mydumper image available